include/qspdlog/qspdlog.hpp

```diff
@@ -105,6 +105,12 @@ class QSpdLog : public QWidget
      */
     std::optional<std::size_t> getMaxEntries() const;
 
+    void setForegroundMapping(const std::string& loggerName, std::optional<QBrush> brush);
+    std::optional<QBrush> getForegroundMapping(const std::string& loggerName) const;
+
+    void setBackgroundMapping(const std::string& loggerName, std::optional<QBrush> brush);
+    std::optional<QBrush> getBackgroundMapping(const std::string& loggerName) const;
```

Comment:
Isn't `(set/get)Logger(Background/Foreground)` better naming?

Reply:
Yes, I just quickly programmed this out and didn't rename it yet.